src/NUnitConsole/nunit4-console/ConsoleOptions.cs

```diff
@@ -68,11 +68,10 @@ public bool WhereClauseSpecified
             get { return WhereClause is not null; }
         }
 
+        public int TestRunTimeout { get; private set; } = -1;
```

Comment:
Would it be reasonably easy to use nullable values for these properties instead of needing a separate `BlahSpecified` property?  Or would that require a bunch of additional work on the args parsing side.